src/compressor/brotli/CMakeLists.txt

```diff
@@ -8,12 +8,12 @@ include(ExternalProject)
 ExternalProject_Add(brotli_ext
   DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
   GIT_REPOSITORY "https://github.com/google/brotli.git"
-  GIT_TAG "master"
-  SOURCE_DIR ${CMAKE_BINARY_DIR}/src/brotli
-  CONFIGURE_COMMAND ./configure-cmake --disable-debug 
+  GIT_TAG "v1.0.7"
+  SOURCE_DIR ${CMAKE_BINARY_DIR}/src//brotli
```

Comment:
why replace `/` with `//` in this line?

Reply:
Sorry, this is a typing error on my part.